### PR TITLE
Remove the DASH documentation format.

### PR DESCRIPTION
**Pull request by gavinbaumanis**
As part of a discussion [here](https://dev.lucee.org/t/documentation-examples/10224/13) @zspitzer suggested that I create a PR with the work required to remove the DASH format of documentation.

This will also negate [LD-123](https://luceeserver.atlassian.net/browse/LD-123).

https://luceeserver.atlassian.net/browse/LD-148

### CODE REVIEW

**Comment by zspitzer** [comment]
@gavinbaumanis if you rev the assets, you need to build and check them in!

however, I don't see any (css/js) changes which require rev-ing the `assetBundleVersion` ?

**Comment by zspitzer** [comment]
@gavinbaumanis let's also remove all the sqlite stuff from the repo as well?

**Comment by gavinbaumanis** [comment]
> @gavinbaumanis let's also remove all the sqlite stuff from the repo as well?

Done.

**Comment by gavinbaumanis** [comment]
> @gavinbaumanis if you rev the assets, you need to build and check them in!
> 
> however, I don't see any (css/js) changes which require rev-ing the `assetBundleVersion` ?

Bumped in error. I have reverted the asset version changes.

File: server/Application.cfc [hunk 1]
```diff
@@ -15,9 +15,9 @@ component {
 	this.mappings[ "/docs"     ] = this.baseDir & "docs";
 	this.mappings[ "/listener" ] = this.baseDir;
 	*/
-	this.assetBundleVersion = 35;  // see parent application.cfc
+	this.assetBundleVersion = 36;  // see parent application.cfc
```
Comment (zspitzer) [inline server/Application.cfc:18]: this is not needed?

**Review by zspitzer — CHANGES_REQUESTED** [review]

File: server/Application.cfc (hunk 1 above)
Comment (gavinbaumanis) [reply server/Application.cfc]: I thought I had already corrected these - sorry.
Done, now!

**Pull request closed without merging**